plugins/json_logs.yaml

```diff
@@ -0,0 +1,50 @@
+version: 0.0.1
+title: JSON
+description: Log parser for JSON
+parameters:
+  - name: log_paths
+    type: "[]string"
+  - name: exclude_log_paths
+    type: "[]string"
+    default: []
+  - name: encoding
+    type: string
+    supported:
+      - nop
+      - utf-8
+      - utf-16le
+      - utf-16be
+    default: utf-8
+  - name: log_type
+    type: string
+    default: json
+  - name: start_at
+    type: string
+    valid_values:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    filelog:
+      include:
+        {{ range $fp := .log_paths }}
+        - '{{ $fp }}'
+        {{end}}
+      start_at: {{ .start_at }}
+      {{ if .exclude_file_log_path }}
```

Comment:
```suggestion
      {{ if .exclude_log_paths }}
```

Reply:
Oops, did not notice that at first. It should be good now though.